src/MixpanelEventForwarder.js

```diff
@@ -183,13 +188,45 @@ var constructor = function () {
         }
     }
 
-    function sendIdentifyRequest(user, userIdentities) {
-        var idForMixpanel;
+    function onIdentifyComplete(user) {
+        // When mParticle identifies a user, the user might
+        // actually be anonymous. We only want to send an
+        // identify request to Mixpanel if the user is
+        // actually known. Only known (logged in) users
+        // will have userIdentities.
```

Comment:
```suggestion
        // Mixpanel considers any user with an identity to be a known user.
        // In mParticle, a user will always have an MPID even if they are anonymous.
        // When mParticle identifies a user, because the user might
        // actually be anonymous, we only want to send an
        // identify request to Mixpanel if the user is
        // actually known. If a user has any user identities, they are 
        // considered to be "known" users.
```